docs/swagger.yml

```diff
@@ -669,6 +669,40 @@ definitions:
         format: date-time
         description: Time when call completed, whether it was successul or failed. Always in UTC.
         readOnly: true
+      stats:
+        type: array
+        items:
+          $ref: '#/definitions/Stat'
+        description: A histogram of stats for a call, each is a snapshot of a calls state at the timestamp.
+        readOnly: true
+
+  Stat:
+    type: object
+    properties:
+      timestamp:
+        type: string
+        format: date-time
+      metrics:
+        type: object
```

Comment:
Omitting mem_limit makes sense. In a previous life this wasn't configured on any user facing struct. Will do.

> net_rx

This isn't true. Many many many many many users will have DB connections, calls to S3 or other services. Absolutely not the case that all communication goes through stdin/stdout, in fact we tell users they must store state elsewhere, which will require a network connection. This information is extremely important to users.

> disk_space 

Users have access to writing to disk. This is unavoidable. The case against this seems vague, at best. It's useful to write to disk in many cases, especially considering that users don't have a lot of ram in this style of programming. Don't really think getting into serverless definition bike shedding is useful. The same could be said of cpu or memory...  Disk is there, users can use it, they could be i/o bound and thus we should expose a way for them to debug that.

Reply:
> Omitting mem_limit makes sense.

I thought about this and, I do think that it would be useful for prometheus to have this since it does not have a handle on a route. one example I can think of is that a UI could show mem % usage. another example would be if there were a bug in the logical memory slot allocator, we could easily see that a node is over allocated vs how much ram it actually has. I do agree that it's somewhat wasteful in how we're tracking it, but it would be nice not to special case things and have the metrics all uniform.